packages/example/scripts/wait-for-postgres-then

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+RETRIES=${RETRIES:-15}
+
+err_file=$(mktemp)
+
+echo "Checking postgres is up..."
+until psql "postgres" -c "select 1" > /dev/null 2> "$err_file" || [ "$RETRIES" -eq 0 ]; do
```

Comment:
Might be better to use [pg_isready](https://www.postgresql.org/docs/11/app-pg-isready.html), which can check for the DB to be created

Reply:
oh nice I never knew about that